docs/usage/initialization.rst

```diff
@@ -101,3 +79,60 @@ Python client initialization:
 
 
 For more information on client side SSL cerificationcas, please refer to this [gist](https://gist.github.com/mtigas/952344).
+
+Dealing with errors during parsing metadata
+-------------------------------------------
+
```

Comment:
In the case where you need to consume a service which has not fully valid metadata document and is  not under your control, you can configure the metadata parser to try to recover from detected problems.

Parser recovery measures include actions such as using a stub entity type, if the parser cannot find a referenced entity type. The stub entity type allows the parser to continue processing the given metadata but causes fatal errors when accessed from the client.